mmedit/models/losses/disco_loss.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
put these three functions in https://github.com/open-mmlab/mmediting/blob/dev-1.x/mmedit/models/losses/pixelwise_loss.py